server/opts.go

```diff
@@ -292,6 +292,7 @@ type Options struct {
 	LogsRetentionDays              int            `json:"-"`
 	TieredStorageUploadIntervalSec int            `json:"-"`
 	DlsRetentionHours              map[string]int `json:"-"`
+	GCProducersConsumersRetention  int            `json:"-"`
```

Comment:
I want people will understand this value is in hours, so only here change to GCProducersConsumersRetentionHours

in case you didn't check that this feature is working when passing a value from the conf file do that and edit the debug/conf files so that they will contain the new parameter